### PR TITLE
bugfix/#4875/_ENDPOINTS]_Refactor_Dto's_bodies_

### PR DESCRIPTION
**Pull request by SokhaTaras**

### CODE REVIEW

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts [hunk 1]
```diff
@@ -83,6 +83,7 @@ export class UbsAdminTariffsDeactivatePopUpComponent implements OnInit, OnDestro
   }
 
   ngOnInit(): void {
+    this.getCouriers();
```
Comment (marianaharnyk) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts]: I guess, that it's bad idea to run this method at the beginning of ngOnInit method. It method getCouriers we use variable currentLanguage, which is defined in 90 line. Therefore in method getCouriers we can get only english name as far as this.currentLanguage === 'ua' will be always false.

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts [hunk 2]
```diff
@@ -91,10 +92,10 @@ export class UbsAdminTariffsDeactivatePopUpComponent implements OnInit, OnDestro
     this.setRegionsPlaceholder();
     this.setCityPlaceholder();
     setTimeout(() => this.city.disable());
-    this.getCouriers();
     this.getReceivingStation();
     this.getLocations();
     this.getTariffCards();
+    const cards = this.tariffCards;
```
Comment (marianaharnyk) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts]: Why do we need this variable?

File: src/app/ubs/ubs-admin/models/tariffs.interface.ts [hunk 3]
```diff
@@ -58,14 +58,16 @@ export interface Stations {
...
   courierId: number;
   courierStatus: string;
-  courierTranslationDtos: CourierTranslationDto[];
+  nameUk: string;
+  nameEn: string;
   createDate: string;
   createdBy: string;
 }
 
 export interface CourierTranslationDto {
```
Comment (marianaharnyk) [inline src/app/ubs/ubs-admin/models/tariffs.interface.ts]: I guess we can remove this interface as far as it is never used

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts (hunk 2 above)
Comment (SokhaTaras) [reply src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts]: already removed

File: src/app/ubs/ubs-admin/models/tariffs.interface.ts (hunk 3 above)
Comment (SokhaTaras) [reply src/app/ubs/ubs-admin/models/tariffs.interface.ts]: already removed

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts [hunk 4]
```diff
@@ -672,7 +664,7 @@ export class UbsAdminTariffsDeactivatePopUpComponent implements OnInit, OnDestro
     return items.filter((option) => option.toLowerCase().includes(filterValue));
   }
 
-  public removeDuplicates(arr: Array<string>): Array<string> {
```
Comment (marianaharnyk) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts]: Why do we need to remove interface?

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts (hunk 1 above)
Comment (SokhaTaras) [reply src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts]: Yeah, you are right, i just was testing some kind of behaviour

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts (hunk 4 above)
Comment (SokhaTaras) [reply src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts]: Already returned)

**Review by atsa21 — APPROVED** [review]

**Review by katia9261 — APPROVED** [review]

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-courier-pop-up/ubs-admin-tariffs-courier-pop-up.component.ts [hunk 5]
```diff
@@ -95,8 +96,8 @@ export class UbsAdminTariffsCourierPopUpComponent implements OnInit, OnDestroy {
   }
 
   checkIsCourierExist(value: string, array: Array<string>): boolean {
-    const newCourierName = value.toLowerCase();
-    const couriersList = array.map((it) => it.toLowerCase());
+    const newCourierName = value;
+    const couriersList = array.map((it) => it);
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-courier-pop-up/ubs-admin-tariffs-courier-pop-up.component.ts]: Why constants needed here?
Remove second parameter for the method

Comment (SokhaTaras) [reply src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-courier-pop-up/ubs-admin-tariffs-courier-pop-up.component.ts]: thanks, fixed

**Review by vplvua — APPROVED** [review]

**Review by AngelinaRudnitska — APPROVED** [review]

**Review by marianaharnyk — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Pull request merged**